target/linux/ramips/mt76x8/target.mk

```diff
@@ -4,7 +4,7 @@
 
 SUBTARGET:=mt76x8
 BOARDNAME:=MT76x8 based boards
-FEATURES+=usb ramdisk small_flash
+FEATURES+=usb ramdisk rtc small_flash
```

Comment:
I would prefer if we do not activate rtc support in this target. Then the RTC chip on your board will not work. 

This will increase the kernel binary for all devices in this target. Some boards do not have much flash.